build.gradle

```diff
@@ -272,3 +274,10 @@ task run(type: Run) {
   group = 'Verification'
   impliesSubProjects = true
 }
+
+task buildSrcIntegTest(type: GradleBuild) {
```

Comment:
I don't think this is necessary, if you just add `integTest` as a dependency of `check` inside `buildSrc/build.gradle`? The `build` task is run on every build, so tests are already run.